site/public/js/simple-grading.js

```diff
@@ -252,6 +252,7 @@ function updateCheckpointCells(elems, scores, no_cookie) {
                     elem.attr('data-score', elem.data('score'));      // update the score
                     elem.attr('data-grader', elem.data('grader'));    // update new grader
                     elem.find('.simple-grade-grader').text(elem.data('grader'));
+
```

Comment:
Are these changes relevant ?

Reply:
No it was from my another PR I have removed them.